dockerfiles/docker-compose.yml

```diff
@@ -162,3 +161,24 @@ services:
     networks:
       readthedocs:
     command: ["server", "--console-address", ":9009", "/data"]
+    
+  createbuckets:
+    image: minio/mc
+    depends_on:
+      - storage
+    environment:
+      - MINIO_ROOT_USER=admin
+      - MINIO_ROOT_PASSWORD=password
+    entrypoint: >
+      /bin/sh -c "
+      /usr/bin/mc alias set myminio http://storage:9000 admin password;
+      /usr/bin/mc mb myminio/static;
+      /usr/bin/mc policy set public myminio/static;
+      /usr/bin/mc mb myminio/media;
+      /usr/bin/mc policy set public myminio/media;
+      /usr/bin/mc mb myminio/build-tools;
+      /usr/bin/mc policy set public myminio/build-tools;
+      exit 0;
+      "
```

Comment:
Why not use the same script that we had for this, `createbucket.sh`, to keep consistency with the rest of the operations?

Reply:
`createbuckets.sh` was executed from the `web` container which doesn't have the mc client.

Reply:
Marking as resolved, LMK if you want a script back. I think that this will impose configuration of the script, that's why I just preferred to express the whole script in .yml rather than adding environment variables or CLI params.